mypyc/genops.py

```diff
@@ -1443,6 +1444,13 @@ def visit_dict_expr(self, expr: DictExpr) -> Value:
                 line=expr.line)
         return dict_reg
 
+    def visit_set_expr(self, expr: SetExpr) -> Value:
+        set_reg = self.add(PrimitiveOp([], new_set_op, expr.line))
```

Comment:
Hm, probably just use `primitive_op` here, for consistency.

`mypyc` is not super great about this sort of consistency but inside one function it's probably worth it.